test/models/bot/smooch_6_test.rb

```diff
@@ -745,4 +750,49 @@ def send_message_outside_24_hours_window(template, pm = nil)
     assert !Bot::Smooch.run(payload.to_json)
     assert_equal '000000:222222', ts.reload.uid
   end
+
+  test 'should not enable NLU for workspace without tipline' do
+    team = create_team
+    assert !Bot::Smooch.enable_nlu(team.slug)
+  end
+
+  test 'should process menu option using NLU' do
```

Comment:
I think this is a good through test! Do we have anything that tests the method that the bot class uses (`nlu_menu_option_from_message`)? If not, could be good to.

Reply:
Yes, under the hood that's what's happening.

Reply:
I think it would to test that method since it's the interface that we use in the implementation - even if it's just a smoke test (though great to have tests around the others if we intend to use them in rails console)

Reply:
Yes, I agree! I just re-implemented `SmoochNlu` as a standalone class so I'm going to add a unit test for that method.